modules/integration/tests-ui-integration/pom.xml

```diff
@@ -243,7 +243,7 @@
     </dependencies>
 
     <properties>
-        <skipUiTests>true</skipUiTests>
+        <skipUiTests>false</skipUiTests>
```

Comment:
@hemikak - You need to enable UI tests though Jenkins. There is a plugin which does the job. Our plan is to skip ui tests from local builds by default. However uses can run ui tests with "mvn install -DskipUiTests=false" command.  Can you check with warunaP on this.